### PR TITLE
#157 report tool version in cli

### LINKED ISSUE #157: --version does not show version
```
$ wget https://github.com/Roave/BackwardCompatibilityCheck/releases/download/4.0.0/roave-backward-compatibility-check.phar

$ php roave-backward-compatibility-check.phar --version
Console Tool
```

### PR DESCRIPTION
**Pull request by Ocramius**
Fixes #157

### CODE REVIEW

File: bin/roave-backward-compatibility-check.php [hunk 1]
```diff
@@ -51,7 +52,7 @@
         throw new RuntimeException('Could not find Composer autoload.php');
     })();
 
-    $application = new Application();
+    $application = new Application('roave/backward-compatibility-check', Versions::getVersion('roave/backward-compatibility-check'));
```
Comment (asgrim) [inline bin/roave-backward-compatibility-check.php]: does this also work in a `Phar` environment?

Comment (Ocramius) [reply bin/roave-backward-compatibility-check.php]: If the phar is built after `composer install` was performed, then yes. The `Versions` class will already have been generated at that point.

Comment (asgrim) [reply bin/roave-backward-compatibility-check.php]: I'd imagine it would be so :+1:

**Comment by asgrim** [comment]
@Ocramius milestone `5.0.0`? could this not be `4.0.1`, as it's not really a BC break, or a feature (unless we already have BC breaks, or new features merged...?)

**Comment by Ocramius** [comment]
@asgrim yeah, for now I'll keep it milestone `5.0.0`, but should we decide to split off `master` now, we can release `5.1.0` with everything that is in that milestone right now.

**Comment by Ocramius** [comment]
Moved to 4.1.0, released.

**Pull request merged**